ocular/src/registry/paths.rs

```diff
@@ -0,0 +1,84 @@
+/// Models for serializing and deserializing IBC path JSON data found in the `_IBC/` directory of the registry repository
+use serde::Deserialize;
+use serde::Serialize;
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase")]
+pub struct IBCPath {
+    #[serde(rename = "$schema")]
+    pub schema: String,
+    #[serde(rename = "chain-1")]
+    pub chain_1: Chain1,
+    #[serde(rename = "chain-2")]
+    pub chain_2: Chain2,
+    pub channels: Vec<Channel>,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase")]
+pub struct Chain1 {
+    #[serde(rename = "chain-name")]
+    pub chain_name: String,
+    #[serde(rename = "client-id")]
+    pub client_id: String,
+    #[serde(rename = "connection-id")]
+    pub connection_id: String,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase")]
+pub struct Chain2 {
+    #[serde(rename = "chain-name")]
+    pub chain_name: String,
+    #[serde(rename = "client-id")]
+    pub client_id: String,
+    #[serde(rename = "connection-id")]
+    pub connection_id: String,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase")]
+pub struct Channel {
+    #[serde(rename = "chain-1")]
+    pub chain_1: Chain12,
+    #[serde(rename = "chain-2")]
+    pub chain_2: Chain22,
+    pub ordering: String,
+    pub version: String,
+    pub tags: Tags,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase")]
+pub struct Chain12 {
+    #[serde(rename = "channel-id")]
+    pub channel_id: String,
+    #[serde(rename = "port-id")]
+    pub port_id: String,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase")]
+pub struct Chain22 {
+    #[serde(rename = "channel-id")]
+    pub channel_id: String,
+    #[serde(rename = "port-id")]
+    pub port_id: String,
+}
```

Comment:
Nit: Potentially renaming these to chain a & b (or some other convention) might be more clear than 1 & 2 as Chain12 & Chain22 read more as chain twelve and chain twenty two imo

Reply:
Unfortunate schema design forced this; I generated the structs with a webtool and it derived those names haha. How about ChainA, ChainAA, ChainB, ChainBB?